WordPress/Classes/ViewRelated/Comments/Views/Detail/CommentContentTableViewCell.swift

```diff
@@ -385,7 +385,6 @@ private extension CommentContentTableViewCell {
         replyButton?.setTitleColor(Style.reactionButtonTextColor, for: .normal)
         replyButton?.setImage(Style.replyIconImage, for: .normal)
         replyButton?.addTarget(self, action: #selector(replyButtonTapped), for: .touchUpInside)
-        replyButton?.flipInsetsForRightToLeftLayoutDirection()
```

Comment:
This had no effect because the buttons are vertical.
The layout is a bit broken. I reported it here https://github.com/wordpress-mobile/WordPress-iOS/issues/23812.

<img width="320" alt="Screenshot 2024-11-14 at 4 19 13 PM" src="https://github.com/user-attachments/assets/b8bb9ee4-8908-4c1c-8726-d14f50821b1c">